stl/inc/regex

```diff
@@ -627,6 +627,11 @@ public:
         return _Compare(_Ptr, _Traits::length(_Ptr));
     }
 
+    void swap(sub_match& _Other) noexcept(_Is_nothrow_swappable<_BidIt>::value) {
+        this->_Mybase::swap(_Other);

```

Comment:
No change requested: As I mentioned on Discord, I think only `_Mybase::swap(_Other);` should be necessary here, but I can't bring myself to reset testing for this, given that it's not harmful and it follows the accepted resolution.